inc/sccm.class.php

```diff
@@ -465,27 +465,49 @@ function getMedias($deviceid, $limit = 99999999) {
    }
 
    static function install() {
-      $cron = new CronTask;
-      if (!$cron->getFromDBbyName(__CLASS__, 'sccm')) {
-         CronTask::Register(__CLASS__, 'sccm', 7 * DAY_TIMESTAMP,
+      $cronCollect = new CronTask;
+
+      if ($cronCollect->getFromDBbyName(__CLASS__, 'sccm')) {
+
+         $cronCollect->fields["name"] = "SCCMCollect";
+         $cronCollect->update($cronCollect->fields);
+
+      } else if (!$cronCollect->getFromDBbyName(__CLASS__, 'SCCMCollect')) {
+
+         CronTask::register(__CLASS__, 'SCCMCollect', 7 * DAY_TIMESTAMP,
             array('param' => 24, 'mode' => CronTask::MODE_EXTERNAL));
+
       }
+
+      CronTask::register(__CLASS__, 'SCCMPush', 7 * DAY_TIMESTAMP,
```

Comment:
Add hour min and hour max to process in this order
1 - Collect
2 - push to fusion